test/components/views/rooms/RoomHeader-test.tsx

```diff
@@ -0,0 +1,252 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import "../../../skinned-sdk";
+
+import * as TestUtils from '../../../test-utils';
+
+import { MatrixClientPeg } from '../../../../src/MatrixClientPeg';
+
+import DMRoomMap from '../../../../src/utils/DMRoomMap';
+import RoomHeader from '../../../../src/components/views/rooms/RoomHeader';
+
+import { Room, PendingEventOrdering, MatrixEvent, MatrixClient } from 'matrix-js-sdk';
+import { SearchScope } from '../../../../src/components/views/rooms/SearchBar';
+import { E2EStatus } from '../../../../src/utils/ShieldUtils';
+import { PlaceCallType } from '../../../../src/CallHandler';
+import { mkEvent } from '../../../test-utils';
+
+describe('RoomHeader', () => {
+    test('shows the room avatar in a room with only ourselves', () => {
```

Comment:
```suggestion
    it('shows the room avatar in a room with only ourselves', () => {
```

I think we usually tend to do this though I don't think it's a requirement

Reply:
Good catch, fixed.